### DISCUSSION TITLE
TIFF conversion to which Pronom ID?

### DISCUSSION THREAD

**Discussion by delta-centauri** [ImageMagick/ImageMagick#4792] [Digital Image Processing]
Hi
I'm using the legacy `convert.exe` from ImageMagick 7.0.11-3 for TIFF conversion using the parameters `-compress none -define tiff:ignore-tags=32934`
The PUID from the source tiff is fmt/353. What will the target be? Can it be changed in the settings somewhere?

**Comment by dlemstra** [comment]
What are: Pronom ID, PUID and fmt/353? I don't understand what you are trying to ask us.

**Reply by delta-centauri** [reply]
From The National Archives:
_PRONOM is an on-line information system about data file formats and their supporting software products. Originally developed to support the accession and long-term preservation of electronic records held by the National Archives, PRONOM is now being made available as a resource for anyone requiring access to this type of information._

The TIFF I used (PUID fmt/353) is described here:
https://www.nationalarchives.gov.uk/PRONOM/fmt/353

I am wondering, which of the TIFF formats is used as the target format from convert.exe. But I guess it is not possible to modify this anyway.
